docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of how Temporal works. 
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
```

Comment:
"Temporal ensures your applications are reliable" <- I feel like this is kind of vague still. If I was coming on this page, had no experience/knowledge of distributed systems, I would be confused by what it means by "makes your applications reliable and handles everything from network outages etc." I would be wondering:

- What makes an application 'reliable'?
- What do you mean by 'handles everything from minor network outages' etc.? 

I would flush out this paragraph a bit more personally!

- What does it mean that

Reply:
I've asked @tomwheeler if he has a script available that you can steal from for his intro to Temporal for onboarding new employees.